### DISCUSSION TITLE
Create an option to have the Fabrica Info dashboard display totals only for DOIs that have been registered, excluding draft state records

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#96] [General Suggestions]
### What is the problem that your suggestion solves?

The Fabrica dashboards on the Info tab display totals that include draft records. This is confusing when creating reports in a consortium context, where the total DOIs registered for a Consortium Organization for the year is the critical metric to capture. 

### What solution might meet your needs?

Create an option to have the dashboard display totals only for DOIs that have been registered, excluding draft state records.

### What alternatives have you tried or considered?

Totaling Registered and Findable counts in the DOIs tab facets for a Consortium Organization.

### Is there anything else you would like to share?

[Link for DataCite staff](https://app.frontapp.com/open/cnv_b8f2dzp?key=piw5SAzi2lvSnxNSj_xP8oh6OsV0xFJj)

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [X] Other

### If other group(s), please describe.

Consortium leads

**Comment by KellyStathis** [comment]
In addition to only showing DOIs in Registered state, it would help to using the date "registered" as opposed to the date "created" to 1) determine which DOIs are shown in the dashboard for a given year and 2) set up the Y-axis in this chart: 
![Screenshot 2024-08-27 at 08 24 06](https://github.com/user-attachments/assets/983cb9b2-70a8-4aa0-8802-b045691b3994)

which also relates to https://github.com/datacite/datacite-suggestions/discussions/45 about y-axis labels
